packages/google_ml_kit/example/lib/nlp_detector_views/entity_extraction_view.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:flutter/material.dart';
 import 'package:google_ml_kit/google_ml_kit.dart';
+import 'package:google_ml_kit_entity_extraction/entity_extractor.dart';
```

Comment:
@bharat-biradar : you do not need this one because it is already exported in `package:google_ml_kit/google_ml_kit.dart`

Reply:
if it is no there, then add it